config/initializers/chartkick.rb

```diff
@@ -151,7 +150,7 @@
          color: 'silver'
       },
       labelStyle: {
-         color: 'silver'
+         color: '#D8D8D8'
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.